src/pages/gateway/command-reference.md

```diff
@@ -48,6 +48,43 @@ The following example creates the environment in the `mesh_examples` subdirector
 Local workspace created successfully
 ```
 
+## aio api-mesh:run
+
+<InlineAlert variant="info" slots="text"/>
+
+This command is currently in beta.
+
+[Deploys a mesh locally](./developer-tools.md#create-a-local-environment). You only need to run this command if you want to work with your mesh locally for testing.
+
+### Usage
+
+```bash
+aio api-mesh:run <port_number>
+```
+
+### Flags
+
+The following arguments are all optional.
+
+`-p` or `--port` allows you to specify the port number for your local environment. The default is `5000`.
+
+`--debug` enters debug mode.
```

Comment:
Hey @jhadobe 

Is it possible to add a note about VS Configuration for debugging?

Here is the config one has to include in their `launch.json` file to quickly debug with the `--debug` argument.

```
{
      "name": "Debug Mesh",
      "port": 9229,
      "request": "attach",
      "skipFiles": ["<node_internals>/**"],
      "type": "node"
    }
```

Reply:
Also, For debug mode :

1. Would it be helpful if we explicitly mention that by default the debug mode is off?
2. Example showing the command to be used to run mesh in debug mode like we have using port